Sources/TokamakCore/Views/DisclosureGroup.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//  Created by Carson Katri on 7/3/20.
+//
+
+public struct DisclosureGroup<Label, Content>: View
+  where Label: View, Content: View {
+  @State public var isExpanded: Bool = false
+  public var isExpandedBinding: Binding<Bool>?
```

Comment:
**nitpick:** To match SwiftUI this should be internal, then exposed via the proxy type. Maybe also `let`? 

```suggestion
  public var isExpandedBinding: Binding<Bool>?
```